x-pack/plugins/observability_solution/entity_manager/server/lib/entities/built_in/services.ts

```diff
@@ -8,61 +8,119 @@
+    filter: '@timestamp >= now-15m',
+    indexPatterns: [
+      'logs-*',
+      'filebeat*',
+      'apm*',
+      'metrics-*',
+      'remote_cluster:logs*',
+      'remote_cluster:filebeat*',
+      'remote_cluster:apm*',
+      'remote_cluster:metrics-*',
+    ],
+    history: {
+      timestampField: '@timestamp',
+      interval: '1m',
+      settings: {
+        frequency: '5m',
+        syncDelay: '5m',
+      },
     },
-    {
-      name: 'errorRate',
-      equation: 'A / 5',
-      metrics: [
-        {
-          name: 'A',
-          aggregation: 'doc_count',
-          filter: 'log.level: "ERROR"',
-        },
-      ],
+    latest: {
+      lookback: '5m',
     },
-  ],
-});
+    identityFields: ['service.name', { field: 'service.environment', optional: true }],
+    displayNameTemplate: '{{service.name}}{{#service.environment}}:{{.}}{{/service.environment}}',
+    metadata: [
+      'data_stream.type',
+      'service.environment',
+      'service.name',
+      'service.namespace',
+      'service.version',
+      'service.runtime.name',
+      'service.runtime.version',
+      'service.language.name',
+      'cloud.provider',
+      'cloud.availability_zone',
+      'cloud.machine.type',
+    ],
+    metrics: [
+      {
+        name: 'latency',
+        equation: 'A',
+        metrics: [
+          {
+            name: 'A',
+            aggregation: 'avg',
+            filter:
+              'processor.event: "metric" AND metricset.name: "transaction" AND data_stream.dataset: "apm.transaction.1m"',
+            field: 'transaction.duration.histogram',
+          },
+        ],
+      },
+      {
+        name: 'throughput',
+        equation: 'A',
+        metrics: [
+          {
+            name: 'A',
+            aggregation: 'doc_count',
+            filter:
+              'processor.event: "metric" AND metricset.name: "transaction" AND data_stream.dataset: "apm.transaction.1m"',
+          },
+        ],
+      },
+      {
+        name: 'failedTransactionRate',
+        equation: 'A / B',
+        metrics: [
+          {
+            name: 'A',
+            aggregation: 'doc_count',
+            filter:
+              'processor.event: "metric" AND metricset.name: "transaction" AND event.outcome: "failure"',
+          },
+          {
+            name: 'B',
+            aggregation: 'doc_count',
+            filter:
+              'processor.event: "metric" AND metricset.name: "transaction" AND event.outcome: *',
+          },
+        ],
+      },
+      {
+        name: 'logErrorRate',
+        equation: 'A / B',
+        metrics: [
+          {
+            name: 'A',
+            aggregation: 'doc_count',
+            filter: 'log.level: "error" OR error.log.level: "error"',
+          },
+          {
+            name: 'B',
+            aggregation: 'doc_count',
+            filter: 'log.level: * OR error.log.level: *',
+          },
+        ],
+      },
+      {
+        name: 'logRate',
+        equation: 'A',
```

Comment:
can you elaborate why 5 is removed from the equation (ex `A/5` ) as it was before?

Reply:
the transform will create a document per bucket with the interval set in `history.interval`. this definition creates buckets of 1m and we'd only need to divide the value if we have 5m buckets